daemons/gptp/linux/src/daemon_cl.cpp

```diff
@@ -278,16 +297,8 @@ int main(int argc, char **argv)
 					return 0;
 				}
 			}
-			else if (strcmp(argv[i] + 1, "V") == 0) {
-				automotive_profile_config.automotiveProfile = true;
-			}
 			else if (strcmp(argv[i] + 1, "ETE") == 0) {
 				automotive_profile_config.automotiveTestMode = true;
-				if (!automotive_profile_config.automotiveProfile) {
-					printf("please use -V option to enable automotive profile first"
-					       "before you change the automotive profile configurations.\n");
-					return 0;
-				}
```

Comment:
I think it might be OK as-is. While most of the automotive features state they are "valid when AVnu Automotive Profile is enabled," test mode just says "as defined in the AVnu automotive profile".